src/nbl/asset/interchange/CPLYMeshFileLoader.cpp

```diff
@@ -57,6 +62,111 @@ bool CPLYMeshFileLoader::isALoadableFileFormat(io::IReadFile* _file) const
     return false;
 }
 
+void CPLYMeshFileLoader::initialize()
+{
+	auto precomputeAndCachePipeline = [&](CPLYMeshFileLoader::E_TYPE type, bool indexBufferBindingAvailable)
```

Comment:
put `IRenderpassIndependentPipelineLoader::initialize();` as the first thing this method does

Reply:
done